components/org.wso2.carbon.identity.oauth.endpoint/pom.xml

```diff
@@ -346,6 +346,7 @@
                 <artifactId>spotbugs-maven-plugin</artifactId>
                 <configuration>
                     <threshold>High</threshold>
+                    <maxHeap>2048</maxHeap>
```

Comment:
Refered from: https://stackoverflow.com/questions/57672501/using-findsecbugs-in-maven-throws-java-lang-outofmemoryerror/57980545#57980545